second/README.md

```diff
@@ -0,0 +1,86 @@
+## Project description
+The project was created in order to share good places with your loved ones.
+## Quick start
+### First step: 
+- Copy project from git
+```
+git clone https://github.com/nester256/rpm-hws_7-1_2023
+
+cd rpm-hws_7-1_2023
+
+git checkout nesterov
+
+cd second
+```
+### Second step:
+- Create docker image 
+```
+docker run  -d \
+        --name restaurants_test \
+        -e POSTGRES_USER=admin \
+        -e POSTGRES_PASSWORD=1234 \
+        -e PGDATA=/postgres_data \
+        -v ~/djangoproject_hw2/postgres_data:/postgres_data \
+        -p 38746:5432 \
+        postgres:15.1
+```
+### Third step:
+```
+./manage.py migrate
```

Comment:
1) we only changed directory to "second" , by manage.py is not there. its in the "restaurants" directory
2) you suggest running manage.py as a script, but it won't run like this as shabang in the beginning says `#!/usr/bin/env python`, and manage.py wont run with python, it must me python3

Reply:
Sorry for stupid mistakes, already corrected